### DISCUSSION TITLE
GSOC 2025

### DISCUSSION THREAD

**Discussion by Ilhan-Personal** [pouriyajamshidi/tcping#277] [General]
Hey @pouriyajamshidi, I was wondering if you thought about maybe proposing tcping in GSOC 2025?

**Comment by pouriyajamshidi** [comment]
Hey @SYSHIL,

I was not aware of that program. Not quite sure what I should do and what it would entail.

Do you have more information in a brief form?

**Reply by Ilhan-Personal** [reply]
Sure, I can try to explain. I'm new to it as well :) 
GSOC is where students or beginners enroll as contributors in open source projects.
Each project will have some mentors who will guide these new contributors.

google will the applications for GSOC 2025 in a couple of months and for tcping to be accepted we'd likely need to apply as a "mentoring organization".
Not sure if adding the project in GSOC will help but I just wanted to send out this info in case it does 😅 

Here are a couple of links to know more about GSOC and the timeline in which it usually operates.
https://summerofcode.withgoogle.com/get-started
https://developers.google.com/open-source/gsoc/timeline

The applications for 2025 have not started, Just sending this out in advance.

**Reply by pouriyajamshidi** [reply]
Thanks a lot for the explanation.

I would love to get more eyes on this project as well. However, 2025 is gonna be a really busy year for me. Not sure how much time I can specify to mentoring. Anyway, I am open to the idea.

**Reply by Ilhan-Personal** [reply]
Sure, makes sense !
Speaking of getting more eyes on this project how do you feel about me writing about it in linkedin, Reddit whenever we make new releases?

**Reply by pouriyajamshidi** [reply]
Sure, that would be great!

I do the same on LinkedIn and X too.

Thanks mate!
